drivers/include/pn532.h

```diff
@@ -0,0 +1,327 @@
+/*
+ * Copyright (C) 2016 TriaGnoSys GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup  drivers_pn532
+ *
+ * @{
+ * @file
+ * @brief   PN532 driver
+ * @author  Víctor Ariño <victor.arino@triagnosys.com>
+ */
+
+#ifndef NFC_READER_INCLUDE_PN532_H_
+#define NFC_READER_INCLUDE_PN532_H_
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include "mutex.h"
+#include "periph/i2c.h"
+#include "periph/gpio.h"
+#include <stdint.h>
+
+/**
+ * @brief Data structure with the configuration parameters
+ */
+typedef struct {
+    i2c_t i2c;                  /** I2C device */
+    gpio_t reset;               /** Reset pin */
+    gpio_t irq;                 /** Interrupt pin */
+} pn532_params_t;
+
+
+/**
+ * @brief Device descriptor for the PN532
+ */
+typedef struct {
+    const pn532_params_t *conf;
+    mutex_t trap;
+} pn532_t;
+
+/**
+ * @brief Internal buffer size
+ *
+ * A small buffer size is enough for most applications, however if large NDEF
+ * files are to be written this size shall be increased. Otherwise the files
+ * can be written in chunks.
+ */
+#ifndef PN532_BUFFER_LEN
+#define PN532_BUFFER_LEN     64
```

Comment:
since you added brackets around constants further below (pn532.c) you could add some here just to keep it consistent.